Makefile

```diff
@@ -10,7 +10,12 @@ gotool:
 	gofmt -w .
 	go vet . | grep -v vendor;true
 build-frontend:
-	cd frontend && npm run build
+	cd frontend && npm run build && cd -
+build: build-frontend
+	rm -rf cmd/fuckdb/cmd/dist
+	cp -r frontend/dist cmd/fuckdb/cmd
+	cd cmd/fuckdb && go build
+
```

Comment:
After build, we should mv `fuckdb` to root dir of project because config file here.
So add `cp cmd/fuckdb/fuckdb .`  after build.